### PR TITLE
outlaw derby page

### PR DESCRIPTION
**Pull request by brian-luoma**
I want to add an outlaw derby page on the pack site that is not the square page.

### CODE REVIEW

File: public/OutlawDerby2024.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+![outlawlogo]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby lOGO.png")
```
Comment (jceaser) [inline public/OutlawDerby2024.md:1]: I would think this would be better to put in the events folder

Comment (jceaser) [inline public/OutlawDerby2024.md:1]: also the direct path to your hard drive will never work on the web, you need a relative URL. The internet also sometimes does not like spaces in file names.

File: public/OutlawDerby2024.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+* 1 PWD car kit
+* Registration
+
+
+
+### [Click here to register](https://cub306.square.site/outlaw-derby-2024)
+![outlaw2024qr]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby qr code.png")
```
Comment (jceaser) [inline public/OutlawDerby2024.md:50]: your C drive is not on the internet

File: public/OutlawDerby2024.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+## Car Creation Guidelines
+**For safety purposes and to protect the race track from damage, cars:**
+
+1) May not interfere with any other cars in the race;
```
Comment (jceaser) [inline public/OutlawDerby2024.md:57]: does ) work for markdown? I guess not. if you switch to '.' then you will get the list styles.

File: public/images/outlaw derby lOGO.png
Comment (jceaser) [inline public/images/outlaw derby lOGO.png:1]: spaces or normally avoided in file names as they require more typing as you either need to use '+' or '%20' depending on context. Just replace with '_'. Same comment for the QR code

Comment (jceaser) [reply public/images/outlaw derby lOGO.png:1]: I would also move both of these to the event folder. They don't need to be in an image directory, that is just convention

**Review by jceaser — CHANGES_REQUESTED** [review]
1. The location of the derby file I disagree with as it litters the root folder.
2. the absolute links to images will not work when published, need to be "relative"
    3. /images/your_pic.png : if you want to reference an image from any page to the top of the web site.
    4. images/your_pic.png : if the image folder is at the same level as your .md file
5. spaces in image names can be annoying in some cases

File: public/OutlawDerby2024.md (hunk 1 above)
Comment (brian-luoma) [reply public/OutlawDerby2024.md]: I created a outlaw folder in the 23-24 events folder and moved the .md and images to that folder

Comment (brian-luoma) [reply public/OutlawDerby2024.md]: im still learning markdown and how to used the editor i'm using

File: public/OutlawDerby2024.md (hunk 2 above)
Comment (brian-luoma) [reply public/OutlawDerby2024.md]: i edited the file paths.

File: public/OutlawDerby2024.md (hunk 3 above)
Comment (brian-luoma) [reply public/OutlawDerby2024.md]: i just copied  and pasted it.

File: public/images/outlaw derby lOGO.png
Comment (brian-luoma) [reply public/images/outlaw derby lOGO.png:1]: I removed the spaces.

**Comment by brian-luoma** [comment]
do i have to close this pull request before i can submit a new pull request so you can see the changes I made?

**Review by jceaser — APPROVED** [review]

**Pull request merged**